### PR TITLE
Feat/multiselect dropdown modal

### PR DESCRIPTION
**Pull request by 0xLodz**
## Purpose
This PR adds the multiselect modal on click on `+n` tag, and display all dropdown options, as shown in the following Figma screenshot.
This modal is part of the multiselect dropdown variant component.
image

## Demo

https://github.com/okp4/dataverse-portal/assets/8344874/00afaafd-97f8-4412-af4c-ab54a3923afd

### CODE REVIEW

File: src/ui/component/dropDown/multiselectDropDown.tsx [hunk 1]
```diff
@@ -1,17 +1,103 @@
...
+            onClick={handleClearAll}
+            variant="quaternary-turquoise"
+          />
+        </>
+      }
+      classes={{
+        main: 'okp4-dataverse-portal-multiselect-dropdown-modal-main'
+      }}
+      closeOnEsc
+      isCentered
```
Comment (AnjaMis) [inline src/ui/component/dropDown/multiselectDropDown.tsx:62]: These two are true by default :)

**Review by AnjaMis — APPROVED** [review]
Looks good to me! 💪🏻

File: src/ui/component/dropDown/multiselectDropDown.tsx (hunk 1 above)
Comment (0xLodz) [reply src/ui/component/dropDown/multiselectDropDown.tsx]: Thanks! updated :)

**Review by ccamel — APPROVED** [review]
LGTM! :+1:

**Pull request merged**